src/components/Card.js

```diff
@@ -20,27 +34,83 @@ export default class Card {
     this._cardImage = this._cardElement.querySelector(".card__image");
     this._cardTitle = this._cardElement.querySelector(".card__title");
     this._likeButton = this._cardElement.querySelector(".card__like-button");
-    this._trashButton = this._cardElement.querySelector(".card__trash-button");
+    this._deleteButton = this._cardElement.querySelector(
+      ".card__delete-button"
+    );
     this._cardImage.src = this._link;
     this._cardImage.alt = this._name;
     this._cardTitle.textContent = this._name;
+    //this._cardElement.id = this.cardId;
     this._setEventListeners();
+    this.renderLikes();
+    this._handleLikeClick();
     return this._cardElement;
   }
-
   _handleLikeButton() {
-    this._likeButton.classList.toggle("card__like-button_active");
+    this._handleCardLike({ _id: this._id, likes: this._likes });
+    this.renderLikes();
+    this._handleLikeClick(this._id);
   }
-  _handleTrashButton() {
-    this._cardElement.remove();
+  toggleLikes() {
+    if (this.isLiked()) {
+      this._cardElement
+        .querySelector(".card__like-button")
```

Comment:
You've found the heart in line 36 as `this._likeButton`, you you need to use this field everywhere


```js
    if (this.isLiked()) {
      this._likeButton.classList.add("card__like-button_active");
    }

```